stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -707,15 +402,18 @@ def _update_plots(self):
         profile_size = 60
 
         rad_prof = self.rad_prof
-        self.out.clear_output(wait=True)
+        self.seeing_profile_plot.clear_output(wait=True)
         ap_settings = ApertureSettings(**self.aperture_settings.value)
-        with self.out:
-            # sub_med += med
+        with self.seeing_profile_plot:
+            r_exact, individual_counts = rad_prof.pixel_values_in_profile
```

Comment:
Is there a reason you moved these variables (`r_exact` and `scaled_exact_counts`) out of the rad_prof instance?